core/generate/mise_step_builder.go

```diff
@@ -29,11 +29,11 @@ type MiseStepBuilder struct {
 	env                   *a.Environment
 }
 
-func (c *GenerateContext) newMiseStepBuilder() *MiseStepBuilder {
+func (c *GenerateContext) NewMiseStepBuilder(displayName string) *MiseStepBuilder {
```

Comment:
The way mise installs packages is by creating a file at `/etc/mise/config.toml`. Since the file is global, there can only be a single mise step builder (which is why it was private and didn't take a name previously)

Reply:
I'm using the default one for build and a new one for runtime dependencies. Is there a better way to do this?